README.md

```diff
@@ -5,6 +5,8 @@
 ## Keysy-peasy
 Keysy-peasy makes keyboard shortcuts easy peasy.
 
+**Please note - Keysy-peasy does not handle browser differences for `key` values, eg, IE uses "Esc" whereas other
```

Comment:
May revisit this later on.

While fully disagree with _keysy-peasy_ having lots of _'if this key also check that or that or that'_ type of logic, when we don't know what the consumer needs to support, however we could potentially update the API and have a separate importable alternative values for compatibility.

Something like: 

```javascript
export interface IShortcut {
    keys: Array<string>;
    callback(event:KeyboardEvent): void ;
    altKey?: boolean;
}
```
and
```javascript
export const escapeCompatibilityKeys = ["Escape", "Esc"];
```

and usage would be something like
```javascript
import { IShortcut, Shortcuts, escapeCompatibilityKeys } from "keysy-peasy";

const shortcuts = new Shortcuts(document);
shortcuts.register([{
   keys: escapeCompatibilityKeys,
   callback: (event) => console.log(event)
}]);
```

Would have to change the registration logic a bit in order to keep event action efficiency, but something like that could keep the compatibility solutions separate from lots of 'if' logic and can be tree-shaken.

Reply:
3 other possible solutions:

1. Allow the use of keyCode field, which is what i'd use for backwards compatibility in this respect
2. Instruct developers to use a polyfill 
3. Add this a as a dep https://github.com/cvan/keyboardevent-key-polyfill/blob/master/index.js as it's pretty smol

 allow the use of a `keyCode` property